configure.ac

```diff
@@ -102,7 +102,6 @@ AC_SUBST([oc_cflags])
 AC_SUBST([oc_cppflags])
 AC_SUBST([oc_ldflags])
 AC_SUBST([bytecclibs])
-AC_SUBST([dllibs])
```

Comment:
I would naively expect this to be replaced by `AC_SUBST([DLLIBS])` rather than removed altogether.